src/EditorFeatures/Test/Diagnostics/IDEDiagnosticIDConfigurationTests.cs

```diff
@@ -710,7 +710,7 @@ public void CSharp_VerifyIDECodeStyleOptionsAreConfigurable()
 No editorconfig based code style option
 
 # IDE0036, PreferredModifierOrder
-csharp_preferred_modifier_order = public,private,protected,internal,static,extern,new,virtual,abstract,sealed,override,readonly,unsafe,volatile,async
+csharp_preferred_modifier_order = public,private,protected,internal,static,extern,new,virtual,abstract,sealed,override,readonly,unsafe,volatile,async:silent
```

Comment:
Did you verify that the setting is actually respected by the IDE? i.e. if you create an editorconfig with the above entry and a custom severity, say error, does the diagnostic show up as error in the IDE?

Reply:
I checked just now and it's respected by Visual Studio.

![image](https://user-images.githubusercontent.com/54114156/66864116-bc02a900-ef94-11e9-86d3-d5727136d24d.png)

Reply:
Thanks, while you are testing this, can you also confirm the "Configure or Suppress issues" group has the "Configure severity" sub-menu, which adds/updates the code style option based entry with severity at the end?

Reply:
"Configure IDE0036 severity" sets the `dotnet_diagnostic.IDE0036.severity` option instead of updating the severity suffix.

Reply:
Thanks for confirming. Would you be able to file a separate tracking issue for it? Tagging @allisonchou - we may have to tweak the configure severity code fix to handle string option values, followed by severity setting.

Reply:
I could file an issue, but debugging this locally this seems to be a simple regex fix (adding a comma to the pattern for allowed option value characters) so if you want I could include it right here and now.

Reply:
That would be great. Thanks @castholm

Reply:
Works as intended now.

![image](https://user-images.githubusercontent.com/54114156/66871029-0f7bf380-efa3-11e9-8346-d4412896107b.png)